src/component/segments/SegmentDocs/SegmentDocs.tsx

```diff
@@ -0,0 +1,89 @@
+import { Alert } from '@material-ui/lab';
+import { useStyles } from 'component/segments/SegmentDocs/SegmentDocs.styles';
+import {
+    STRATEGY_SEGMENTS_LIMIT,
+    SEGMENT_VALUES_LIMIT,
+} from 'utils/segmentLimits';
+
+export const SegmentDocsWarning = () => {
+    const styles = useStyles();
+
+    return (
+        <Alert severity="warning">
+            <p className={styles.paragraph}>
+                Segments is an experimental feature available to select users.
+            </p>
+            <p className={styles.paragraph}>
+                This feature is currently in development and future versions may
```

Comment:
Minor phrasing suggestion. I feel like this would read better 💁🏼

```suggestion
                This feature is currently in development. Future versions may
```

Reply:
True, much better!